README_ja-JP.md

```
@@ -0,0 +1,335 @@
+
+**PUBG**
+
+On a high level it’s the same as Antutu, but some slight differences that only benefit games. Think of network prioritization and touch input latency improvements. The manufacturers goal when they detect Antutu is **ALL THE POWER**. Benchmarks are relatively short and it makes them look better on comparison websites.
+
+The goal for PUBG is more like **MORE POWER**, since the intention is often to have a game running for longer it has less aggressive changes.
+
+**TLDR and summary:**
+
+**Antutu** spoofing:
+
+    •    Maximum CPU/GPU frequencies unlocked
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)